### PR TITLE
Cache Manager: Fix code style issues (2/2)

### PR DESCRIPTION
**Pull request by sjinks**
## Description

This PR fixes all code style issues that required manual intervention. It is intended to be merged into the feature branch (see #2428). I have split these PRs so that it is more convenient to review them.

**NOTE:** this PR should be merged after #2428 is reviewed and approved.

### Cache Manager

Fix code standards compliance issues

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

None

### CODE REVIEW

File: vip-cache-manager.php [hunk 1]
```diff
@@ -8,5 +8,5 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
-require_once __DIR__ . '/vip-cache-manager/vip-cache-manager.php';
+require_once __DIR__ . '/vip-cache-manager/class-wpcom-vip-cache-manager.php';
```
Comment (sjinks) [inline vip-cache-manager.php:11]: > Class file names should be based on the class name with "class-" prepended. Expected class-wpcom-vip-cache-manager.php, but found vip-cache-manager.php. (WordPress.Files.FileName.InvalidClassFileName)

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 2]
```diff
@@ -400,12 +419,14 @@ public function execute_purges() {
 		$num_ban_urls   = count( $this->ban_urls );
 		$num_purge_urls = count( $this->purge_urls );
 		if ( $num_ban_urls > self::MAX_BAN_URLS ) {
-			trigger_error( sprintf( 'vip-cache-manager: Trying to BAN too many URLs (total count %s); limiting count to %d', number_format( $num_ban_urls ), self::MAX_BAN_URLS ), E_USER_WARNING );
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+			trigger_error( sprintf( 'vip-cache-manager: Trying to BAN too many URLs (total count %s); limiting count to %d', number_format( $num_ban_urls ), (int) self::MAX_BAN_URLS ), E_USER_WARNING );
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:423]: I have added `(int)` to `self::MAX_BAN_URLS` to suppress the `WordPress.Security.EscapeOutput.OutputNotEscaped` error. phpcs does not check the format string and therefore generates false-positives.

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 3]
```diff
@@ -419,24 +440,23 @@ public function execute_purges() {
...
-	public function purge_site_cache( $when = null ) {
+	public function purge_site_cache() {
 		if ( $this->site_cache_purged ) {
 			return;
 		}
 
 		$this->ban_urls[]        = untrailingslashit( home_url() ) . '/(?!wp\-content\/uploads\/).*';
 		$this->site_cache_purged = true;
-
-		return;
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:439]: > Empty return statement not required here (Squiz.PHP.NonExecutableCode.ReturnNotRequired)

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 4]
```diff
@@ -419,24 +440,23 @@ public function execute_purges() {
...
 
 		if ( empty( $requests ) ) {
 			return;
 		}
 
-		return $this->curl_multi( $requests );
+		$this->curl_multi( $requests );
 	}
 
-	public function purge_site_cache( $when = null ) {
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:431]: SonarLint complained that `$when` is never used.

I have checked that no call site passes parameters to `purge_site_cache` anyway.

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 5]
```diff
@@ -419,24 +440,23 @@ public function execute_purges() {
 			$requests = array_merge( $requests, $this->build_purge_request( $url, 'PURGE' ) );
 		}
 
-		$this->ban_urls = $this->purge_urls = array();
+		$this->ban_urls   = [];
+		$this->purge_urls = [];
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:444]: > Assignments must be the first block of code on a line (Squiz.PHP.DisallowMultipleAssignments.Found)

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 6]
```diff
@@ -456,12 +476,12 @@ public function queue_post_purge( $post_id ) {
 		}
 
 		if ( ! is_post_type_viewable( $post->post_type ) ) {
-			return;
+			return false;
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:479]: Here (and below) SonarLint complained that the `return` statement is not used consistently — it should either return nothing everywhere or return a boolean.

File: vip-cache-manager/class-wpcom-vip-cache-manager.php [hunk 7]
```diff
@@ -754,7 +774,8 @@ public function queue_purge_url( $url ) {
 		$is_valid_url   = $this->is_valid_purge_url( $normalized_url );
 
 		if ( false === $is_valid_url ) {
-			trigger_error( sprintf( 'vip-cache-manager: Tried to PURGE invalid URL: %s', $url ), E_USER_WARNING );
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+			trigger_error( sprintf( 'vip-cache-manager: Tried to PURGE invalid URL: %s', esc_html( $url ) ), E_USER_WARNING );
```
Comment (sjinks) [inline vip-cache-manager/class-wpcom-vip-cache-manager.php:778]: > All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$url'. (WordPress.Security.EscapeOutput.OutputNotEscaped)

File: vip-cache-manager.php (hunk 1 above)
Comment (WPprodigy) [reply vip-cache-manager.php:11]: I'm not sure if this is a standard we wish to adhere to.

Comment (sjinks) [reply vip-cache-manager.php:11]: No problem, should we add an exclusion to phpcs.xml?

Comment (rinatkhaziev) [reply vip-cache-manager.php:11]: > No problem, should we add an exclusion to phpcs.xml?

Yes, please

Comment (sjinks) [reply vip-cache-manager.php]: Fixed in 8d5b9a8e2

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**